src/frontend/komponenter/Fagsak/Fakta/FaktaSkjema.tsx

```diff
@@ -1,215 +1,265 @@
-                stegtekst={actionBarStegtekst(Behandlingssteg.Fakta)}
-                forrigeAriaLabel={behandling.harVerge ? 'Gå tilbake til vergesteget' : undefined}
-                nesteAriaLabel="Gå videre til foreldelsessteget"
-                onForrige={behandling.harVerge ? gåTilForrige : undefined}
-                onNeste={sendInnSkjema}
-                isLoading={senderInn}
-            />
-        </HGrid>
+                    <Textarea
+                        label="Hvordan ble feilutbetalingen oppdaget?"
+                        {...methods.register('vurdering.oppdaget.beskrivelse')}
+                        size="small"
+                        className="w-100 mb-6"
+                        minRows={3}
+                        resize
+                        maxLength={3000}
+                    />
+                </section>
+                <ActionBar
+                    stegtekst={actionBarStegtekst(Behandlingssteg.Fakta)}
+                    forrigeAriaLabel={undefined}
+                    nesteAriaLabel="Gå videre til foreldelsessteget"
+                    onForrige={undefined}
+                    onNeste={() => {}}
+                    isLoading={false}
+                />
+            </form>
+        </FormProvider>
     );
 };
 
-export default FaktaSkjema;
+const PeriodeRad = ({
+    periode,
+    periodeIndex,
+    periodeInfo,
+    muligeRettsligGrunnlag,
+}: {
+    periode: OppdaterFaktaPeriodeDto;
+    periodeIndex: number;
+    periodeInfo: FaktaPeriodeDto;
+    muligeRettsligGrunnlag: MuligeRettsligGrunnlagDto[];
+}): React.JSX.Element => {
+    const tilgjengeligeGrunnlag = (bestemmelse: string): BestemmelseEllerGrunnlagDto[] =>
+        muligeRettsligGrunnlag.find(
+            muligGrunnlag => muligGrunnlag.bestemmelse.nøkkel == bestemmelse
+        )?.grunnlag ?? [];
+    const { register } = useFormContext<OppdaterFaktaOmFeilutbetalingDto>();
+    return (
+        <Table.Row>
+            <Table.DataCell>
+                <span className="ml-2">
+                    {formatterDatostring(periodeInfo.fom)}–{formatterDatostring(periodeInfo.tom)}
+                </span>
+            </Table.DataCell>
+            <Table.DataCell>
+                <VStack>
+                    {periode.rettsligGrunnlag.map((_, index) => (
+                        <Select
+                            label="Velg bestemmelse"
+                            hideLabel
+                            size="small"
+                            key={index}
+                            {...register(
+                                `perioder.${periodeIndex}.rettsligGrunnlag.${index}.bestemmelse`
+                            )}
+                            className="flex-1"
+                        >
+                            <option value="default" disabled>
+                                Velg bestemmelse
+                            </option>
+                            {muligeRettsligGrunnlag.map(grunnlag => (
```

Comment:
Hvis du destructer denne  og fjerner grunnlag under så får vi <option>-tagen over 3 linjer i stedet for 6 linjer.
```suggestion
                            {muligeRettsligGrunnlag.map(({ bestemmelse }) => (
```